msg/messages.js

```diff
@@ -975,6 +975,23 @@ Blockly.Msg.LISTS_GET_SUBLIST_TAIL = '';
 /// [[File:Blockly-get-sublist.png]]
 Blockly.Msg.LISTS_GET_SUBLIST_TOOLTIP = 'Creates a copy of the specified portion of a list.';
 
+/// url - Infomation describing sorting a list.
+Blockly.Msg.LISTS_SORT_HELPURL = 'https://github.com/google/blockly/wiki/Lists#sorting-a-list';
+/// Sort a list of items.
```

Comment:
There is no information here for the translators to know what to do with %1 %2 %3.  The Hungarian translator may need to change this to "%1 be sorted %3 %2".